ci-operator/step-registry/multiarch/nodes-readiness/multiarch-nodes-readiness-commands.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ -z "${CONTROL_ARCH}" ] || [ -z "${COMPUTE_ARCH}" ]; then
```

Comment:
Shouldn't this be:
```suggestion
if [ -z "${CONTROL_ARCH}" ] && [ -z "${COMPUTE_ARCH}" ]; then
```

Reply:
I guess it doesn't really matter since multi arch clusters will need both defined?

Reply:
Yea I was undecided too. What do we prefer? This will depend on how these veriables will evolve and how we will unify/refactor the OCP_ARCH and affiliated ones

Reply:
I think we can leave as is and revisit as we make the usage more consistent.

Reply:
+1